README.md

```diff
@@ -140,14 +236,62 @@ Imagine that a deck of playing cards is sorted by rank and suit: first all the A
 
```

Comment:
One point of note before getting to your answers here: Remember that, when cloning your git repository, to create your own folder to store your homework in, rather than directly editing the source material. In addition to this, please create a new file (or edit the template included) rather than editing the readme file.